### PR TITLE
feat: orderstatus integrated fullwidthdivider component

### PR DESCRIPTION
**Pull request by kibo-sushant**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,11 @@
+KIBO_AUTH_HOST=home.mozu.com
```
Comment (kibo-chandradeeptalaha) [inline .env:1]: This file shouldn't be here. Change in .env.local

File: components/common/FilterOrders/FilterOrders.tsx [hunk 2]
```diff
@@ -11,6 +11,7 @@ const styles = {
     color: '#2b2b2b',
```
Comment (kibo-chandradeeptalaha) [inline components/common/FilterOrders/FilterOrders.tsx:11]: use color from theme

File: components/order-status/ViewOrderStatus/ViewOrderStatus.tsx [hunk 3]
```diff
@@ -123,7 +140,7 @@ const ViewOrderStatus = (props: ViewOrderStatusProps) => {
 
       {lookupErrorMessage && <Box>{lookupErrorMessage}</Box>}
```
Comment (kibo-chandradeeptalaha) [inline components/order-status/ViewOrderStatus/ViewOrderStatus.tsx:141]: Aren't we gonna show this in some red color text ?

**Review by kibo-chandradeeptalaha — CHANGES_REQUESTED** [review]
Please take a look

File: components/common/FilterOrders/FilterOrders.tsx (hunk 2 above)
Comment (kibo-sushant) [reply components/common/FilterOrders/FilterOrders.tsx:11]: As mentioned in code, it's dummy implementation

**Review by kibo-chandradeeptalaha — APPROVED** [review]

**Review by kibo-kevinwatts — APPROVED** [review]

**Pull request merged**